tv_apps/remote-control/js/config_section.js

```diff
@@ -0,0 +1,127 @@
+/* global Section, Settings */
+'use strict';
+
+(function(exports) {
+  function ConfigSection(id) {
+    Section.call(this, id);
+    Settings.on('changed', this._onSettingsChanged.bind(this));
+  }
+
+  ConfigSection.prototype = Object.create(Section.prototype);
+
+  // Section's Interface
+  ConfigSection.prototype._readyToShow = function() {
+    var checkedOption;
+    if (Settings['remote-control.enabled']) {
+      if (Settings['remote-control.pairing-required']) {
+        checkedOption = 'option-pair-on';
+      } else {
+        checkedOption = 'option-pair-off';
+      }
+    } else {
+      checkedOption = 'option-disabled';
+    }
+
+    this.checkRadio(checkedOption);
+    this.setDefaultFocusElement(document.getElementById(checkedOption));
+
+    this.updateClearPairedDevicesButton();
+  };
+
+  // Section's Interface
+  ConfigSection.prototype._handleClick = function() {
+    if (this._pendingSave) {
+      return;
+    }
+
+    var elem = this.getFocusedElement();
+    switch(elem.id) {
+      case 'option-pair-off':
+      case 'option-pair-on':
+      case 'option-disabled':
+        this.checkRadio(elem.id);
+        break;
+      case 'clear-paired-devices':
```

Comment:
I'm confused by your interchanging term "paired device" and "authorized device". Could we unify them?